src/common/components/Button/Button.tsx

```diff
@@ -7,15 +7,16 @@ export interface IButtonProps {
   type?: 'submit' | 'button'
   style?: CSSProperties
   onClick?: () => void
+  children: React.ReactNode
 }
 
-const Button: React.FC<IButtonProps> = ({
+function Button({
```

Comment:
Please use the updated button and Input components. You can pull it from new_development branch